test/analysis_ee_ZH_cfg.py

```diff
@@ -226,6 +236,7 @@ def relative_isolation(lepton):
 
     import random
     random.seed(0xdeadbeef)
+    pdebug.open("/Users/alice/work/Outputs/pythonphysics.txt")
```

Comment:
probably needs to be moved to the module, out of the main section. 
initialize the pdebug module (with the logger) next to the main logger